settings.d/remote_execution_ssh.yml.example

```diff
@@ -3,3 +3,4 @@
 :ssh_identity_key_file: '~/.ssh/id_rsa_foreman_proxy'
 :local_working_dir: '/var/tmp'
 :remote_working_dir: '/var/tmp'
+:kerberos_auth: false
```

Comment:
A hint: if we keep the setting commented, we have more control over the default setting in the code, if we wanted to enable it by default in future for some reason.